### PR TITLE
add otel.status_code and otel.status_message tags for Zipkin exporter

### PR DESCRIPTION
**Pull request by danrusei**
to address #223

### CODE REVIEW

File: opentelemetry-zipkin/src/model/mod.rs [hunk 1]
```diff
@@ -25,6 +25,29 @@ impl Into<annotation::Annotation> for api::Event {
...
+        api::StatusCode::PermissionDenied => "PermissionDenied",
+        api::StatusCode::ResourceExhausted => "ResourceExhausted",
+        api::StatusCode::FailedPrecondition => "FailedPrecondition",
+        api::StatusCode::Aborted => "Aborted",
+        api::StatusCode::OutOfRange => "OutOfRange",
+        api::StatusCode::Unimplemented => "Unimplemented",
+        api::StatusCode::Internal => "InternalError",
+        api::StatusCode::Unavailable => "Unavailable",
+        api::StatusCode::DataLoss => "DataLoss",
+        api::StatusCode::Unauthenticated => "Unauthenticated",
```
Comment (frigus02) [inline opentelemetry-zipkin/src/model/mod.rs:47]: The [spec](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/sdk_exporters/zipkin.md) mentions that the status code should follow the [gRPC spec](https://github.com/grpc/grpc/blob/master/doc/statuscodes.md), which formats them in uppercase. I assume that's what we should do as well, but I'm not too familiar with Zipkin. What do you think?

**Review by frigus02 — COMMENTED** [review]
Thanks for the PR. This looks great.

File: opentelemetry-zipkin/src/model/mod.rs (hunk 1 above)
Comment (danrusei) [reply opentelemetry-zipkin/src/model/mod.rs:47]: Good point. The GO implementation is using [codes](https://pkg.go.dev/google.golang.org/grpc/codes). The differences I noticed are on "UnknownError" vs "Unknown"  and " InternalError" vs "Internal".

Comment (frigus02) [reply opentelemetry-zipkin/src/model/mod.rs:47]: I just had a look at the other SDKs:

- The JavaScript SDK uses uppercase status code names. See [Zipkin exporter](https://github.com/open-telemetry/opentelemetry-js/blob/f731c65201ef49d91d8ec14ad329d9e485e2bcfb/packages/opentelemetry-exporter-zipkin/src/transform.ts#L81) and [CanonicalCode declaration](https://github.com/open-telemetry/opentelemetry-js/blob/f731c65201ef49d91d8ec14ad329d9e485e2bcfb/packages/opentelemetry-api/src/trace/status.ts).
- The Java SDK uses uppercase status code names, but they already implemented the status change away from gRPC status code equivalents to just unset, ok, error. See [Zipkin exporter](https://github.com/open-telemetry/opentelemetry-java/blob/64c2b8cc65a023bdf6f1e09bfc73bbe20168b627/exporters/zipkin/src/main/java/io/opentelemetry/exporters/zipkin/ZipkinSpanExporter.java#L157) and [CanonicalCode declaration](https://github.com/open-telemetry/opentelemetry-java/blob/64c2b8cc65a023bdf6f1e09bfc73bbe20168b627/api/src/main/java/io/opentelemetry/trace/Status.java#L45).
- And the Go SDK uses numbers?

Comment (danrusei) [reply opentelemetry-zipkin/src/model/mod.rs]: I guess you are right, this is not a long lasting change as the spec has changed. GO impl. has an open PR to migrate to new codes.

**Review by frigus02 — APPROVED** [review]

**Pull request merged**